client/index.jsx

```diff
@@ -0,0 +1,30 @@
+import $ from 'jquery';
+
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data:[]
+    }
+  }
+
+  componentDidMount() {
+    $.get({
```

Comment:
Suggestion: you might consider using a different request than jQuery AJAX unless you plan on using other jQuery functions. Otherwise you import the entire jQuery library just to use one function. Doesn't make any functional difference though, just a potential size/efficiency improvement.